common/templates/partials/fs-cookies-start.html

```diff
@@ -0,0 +1,89 @@
+
+{% set acceptHtml %}
+<p class="govuk-body">{% trans %}You've accepted analytics cookies. You can{% endtrans %} <a class="govuk-link"
+        href="{{ url_for('content_routes.cookie_policy') }}">{% trans %}change your cookie settings{% endtrans %}</a> {%
+    trans %}at any time.{% endtrans %}</p>
+{% endset %}
+
+{% set denyHtml %}
+<p class="govuk-body">{% trans %}You've rejected analytics cookies. You can{% endtrans %} <a class="govuk-link"
+        href="{{ url_for('content_routes.cookie_policy') }}">{% trans %}change your cookie settings{% endtrans %}</a> {%
```

Comment:
How this HTML is laid out is a little chaotic for me - I don't want this to block anything but whatever is formatting this and deciding to break {% %} on to new new lines needs to stop?

Reply:
agree but until we find a functional auto-formatter for jinja then I think we just have to suck this up a bit.